jetson/percep/pcl.hpp

```diff
@@ -64,6 +72,12 @@ class PCL {
 
     };
 
+    //Destructor for PCL
+    ~PCL() {
+        viewer -> close();
```

Comment:
This would be a good place for those #if statements

Reply:
Fixed